circle.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
 
 test:
   override:
-    - shellcheck bin/docker-* bin/ensure-kubectl bin/k8s-apply bin/k8s-secrets-from-s3 bin/verify-deployment bin/k8s-lint
+    - shellcheck bin/docker-* bin/ensure-kubectl bin/k8s-apply bin/k8s-secrets-from-s3 bin/verify-deployment bin/k8s-lint bin/cloud-login
```

Comment:
I had added those scripts to the shell check variables in the `eval` lines in `prepare-awscli`  threw errors I couldn't work around.